modules/ROOT/pages/govern-api-instances.adoc

```diff
@@ -0,0 +1,45 @@
+= Governing API Instances
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+
+API Manager is closely integrated with xref:api-governance::index.adoc[API Governance] to govern aspects of API instances with governance rulesets. Applying a governance ruleset to your API ensures that every instance of that API conforms to standards such as what policies are applied and if a TLS context is required.  
+
+To govern an API instances, see <<apply>>.
+
+After applying a Governance ruleset to a specific API, view and fix the conformance issues for its API instances:
+
+* <<view-api-manager>>
+* <<view-api-governance>>
+
+[[apply]]
+== Apply a Governance Rulesets
+
+To govern API instances, you must apply a governance ruleset to your desired API. To apply a governance ruleset, see xref:api-governance::create-profiles.adoc[].
+
+Additionally, you can create custom rulesets to satisfy your specific needs. To create custom governance rulesets, see xref:api-governance::create-custom-rulesets.adoc[].
+
+[[view-api-manager]]
+
+== View and Fix Instance Conformance issues in API Manager
+
+To view the conformance status of an API instance:
+
+. Navigate to *Anypoint Platform* > *API Manager*. 
+. Click the name of the API instance whose conformance status you want to view.
++
+*Instance Conformace* appears in the *API Summary* pane.
+
+. Click *Governance Report* to view the individual rules the instance breaks.
```

Comment:
```suggestion
. Click *Governance Report* to view the individual rules the instance fails.
```

Reply:
It's not exactly showing the individual rule level, as it is a bit of a rollup. Unless they view the ruleset, it's not at the rule level, so at this point it is best to talk about the details for each ruleset (or you could say issues for the rulesets the instance is nonconformant to)